Directory.Packages.props

```diff
@@ -6,7 +6,10 @@
     <PackageVersion Include="Microsoft.Maui.Controls.Compatibility" Version="8.0.20" />
```

Comment:
Maybe not force higher version of Maui nugets than those included in latest workload. I think they are at 8.0.7

Reply:
I recommend using the latest ones as there are quite a lot of bugfixes since the 8.0.7 version (8.0.20 is 3 servicing releases newer than 8.0.7). If you want to use a different version of the MAUI nugets, that's also good, but let's do that in a separate PR? (this only touches the Rive version and its dependencies, I didn't touch the MAUI version)